### PR TITLE
Mr kck

### PR DESCRIPTION
**Pull request by TheMLGuy**
Check this out
Keras is an awesome library https://keras.io
Pls share your insights
Run using python pokemon.py

### CODE REVIEW

**Comment by erilyth** [comment]
@Mr-Kickass 
1) Yes, Keras is a great library but the aim of this challenge was to learn how to use Tensorflow directly. There are certain advantages we have when we use these underlying libraries like Tensorflow or Theano directly without using higher level wrappers around them like Keras or Lasagne.
2) The code you wrote in Keras seems great! Well done. However do make sure that your test data and train data do not overlap. In your case they do which means you can over-fit the network to the training data and then you would end up getting 100% accuracy on your test data.

**Comment by TheMLGuy** [comment]
Hey pal,
Yes it does overlap
However, this is just version 1. I hope to review and change most of it.
Thanks
Ashwin
